frontend/src/pages/projectPage/ProjectList.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import ProjectModal from './ProjectModal';
+import TaskList from './TaskList';
+import './ProjectList.css';
+import image1 from '../../assets/images/Rectangle 2996.jpg';
+import image2 from '../../assets/images/Rectangle 2997.jpg';
+import image3 from '../../assets/images/Rectangle 2998.jpg';
+import image4 from '../../assets/images/Rectangle 2999.jpg';
```

Comment:
how are you planning on managing these assets in the future? Is the user supposed to upload their own project images or choose from these ones?

Reply:
I'm actually not sure about this, I have to speak with my team mates :)